app/scripts/modules/core/src/loadBalancer/LoadBalancerSearchResultFormatter.ts

```diff
@@ -16,11 +14,14 @@ export class LoadBalancerSearchResultFormatter implements ISearchResultFormatter
 
   public get displayName() { return 'Load Balancers'; }
   public get order() { return 5; }
-  public get iconClass() { return 'icon icon-elb'; }
+  public get iconClass() { return 'icon icon-elb'; }  // TODO: ANG replace w/ fa-sitemap
```

Comment:
any reason not to do this now?

Reply:
nope.  done.  there's gobs of other usages in the code for icon/icon-elb we should pull as well.